src/util/message.cpp

```diff
@@ -14,7 +15,27 @@
 #include <string>
 #include <vector>
 
-const std::string strMessageMagic = "Bitcoin Signed Message:\n";
+static const std::string MESSAGE_MAGIC = "Bitcoin Signed Message:\n";
+
+bool MessageSign(
+    const CKey& privkey,
+    const std::string& message,
+    std::string* signature)
```

Comment:
This should be a reference (`std::string&`) instead of pointer.

Reply:
Done.